docs/limitations.rst

```diff
@@ -52,7 +52,7 @@ clean, well written patch would likely be accepted to solve them.
 
 - The ``cpptest`` does not run on Windows with Python 3.8 or newer, due to DLL
```

Comment:
Oh, that limitation does not exist anymore:

https://github.com/pybind/pybind11/pull/5179/files

We discovered that more-or-less by luck and forgot to remove the bullet point here.

Since you opened this PR already, could you please change it to remove the bullet point entirely and reference PR #5179 in your PR description?

@henryiii for awareness.

Reply:
Done.